doc/proxy-clients.rst

```diff
@@ -97,12 +97,28 @@ You can also pass some options to the Varnish client:
 * ``tags_header`` (X-Cache-Tags): Allows you to change the HTTP header used for
   tagging. If you change this, make sure to use the correct header name in your
   :doc:`proxy server configuration <proxy-configuration>`;
+* ``tags_purge_header`` (key): Allows you to change the HTTP header used for
+  purging tags when ``tag_mode`` is set to one of the ``purge*`` options. If
+  you change this, make sure to use the correct header name in your
+  :doc:`proxy server configuration <proxy-configuration>`;
+* ``tag_mode`` (ban): Makes it possible to invalidate tags
+  using purge when for instance using ``xkey`` instead of ban. Supports three
+  modes: ``ban``, ``purge`` and ``purge_single`` for purge per tag.
```

Comment:
hm. should we remove purge for now and only have purge_single? if the feature is not even merged yet, its unsure if it ever will or it might be different. and if we only add it after its released, we can specify the version of the vmod that you need